### PR TITLE
Fixes #2652:  Add missing presets to frskyDSetDefault()

### LINKED ISSUE #2652: Add missing presets to frskyDSetDefault()
The `frskyDSetDefault()` is missing some presets that are defined in `frskySportSetDefault()`. Related conversation in https://github.com/opentx/opentx/issues/2646#issuecomment-128309838

### PR DESCRIPTION
**Pull request by projectkk2glider**
Closes #2652

### CODE REVIEW

File: radio/src/telemetry/frsky_sport.cpp [hunk 1]
```diff
@@ -267,6 +267,7 @@ void frskySportSetDefault(int index, uint16_t id, uint8_t instance)
     uint8_t prec = min<uint8_t>(2, sensor->prec);
     telemetrySensor.init(sensor->name, unit, prec);
     if (id == RSSI_ID) {
+      telemetrySensor.filter = 1;
```
Comment (kilrah) [inline radio/src/telemetry/frsky_sport.cpp]: I remember filter was disabled by default on purpose for smart port telemetry. We receive too few samples in sport and it was introducing too much lag.

Comment (projectkk2glider) [inline radio/src/telemetry/frsky_sport.cpp]: I wondered about that, will fix immediately.

**Comment by bsongis** [comment]
Yes, thanks, I forgot them.
I think that the conversion m=>ft according to the imperial parameter should not be done here, but in telemetry.cpp instead (it's common to all telemetry protocols)

**Comment by kilrah** [comment]
Correct!

**Pull request merged**